tests/data/edlifo-project/Scenarios/s0/PostProcess/curtailment_report_0.0449_Yes_pctpmpp.csv

```diff
@@ -0,0 +1,97 @@
+,pvgui_mpx000460267NetCurt,pvgui_mpx000460267kWCurt,pvgui_mpx000460267PctCurt,pvgui_mpx000460267PctPmpp,pvgui_mpx000637601NetCurt,pvgui_mpx000637601kWCurt,pvgui_mpx000637601PctCurt,pvgui_mpx000637601PctPmpp,pvgnem_mpx000594341NetCurt,pvgnem_mpx000594341kWCurt,pvgnem_mpx000594341PctCurt,pvgnem_mpx000594341PctPmpp,pvgnem_mpx000460267NetCurt,pvgnem_mpx000460267kWCurt,pvgnem_mpx000460267PctCurt,pvgnem_mpx000460267PctPmpp,pvgnem_mpx000635970NetCurt,pvgnem_mpx000635970kWCurt,pvgnem_mpx000635970PctCurt,pvgnem_mpx000635970PctPmpp,pvgui_mpx000460267Ref,pvgui_mpx000460267Actual,pvgui_mpx000460267kVarActual,pvgui_mpx000637601Ref,pvgui_mpx000637601Actual,pvgui_mpx000637601kVarActual,pvgnem_mpx000594341Ref,pvgnem_mpx000594341Actual,pvgnem_mpx000594341kVarActual,pvgnem_mpx000460267Ref,pvgnem_mpx000460267Actual,pvgnem_mpx000460267kVarActual,pvgnem_mpx000635970Ref,pvgnem_mpx000635970Actual,pvgnem_mpx000635970kVarActual,Pct_kW_curtailment,Total_PV_kW_deployed,Total_PV_kW_curtailed,Total_PV_kW_output,Comment
```

Comment:
These output files should also not be part of the repo, right?

Reply:
Okay. I just uploaded the project folder as I tested it.